dockerfiles/launcher/launcher_funcs.sh

```diff
@@ -166,6 +166,22 @@ docker_run_with_debug() {
   fi
 }
 
+docker_run_with_volumes_lib() {
+  if has_lib_volume; then
+    docker_run_with_debug -v "${CHE_LIB_VOLUME}":/data/lib "$@"
```

Comment:
Ah, no problem, I should have checked them first :)  I'm looking at the scripts now.